verification/client.go

```diff
@@ -192,6 +197,41 @@ func (c *Client[CurveParameter, Digest]) CertifyKey(cmd *CertifyKeyReq[Digest])
 	}, nil
 }
 
+// GetCertificateChain calls the DPE GetCertificateChain command.
+func (c *Client[_, _]) GetCertificateChain(cmd *GetCertificateChainReq) (*GetCertificateChainResp, error) {
```

Comment:
The goal of this function is just to get the full cert chain, so I don't think you need to pass in GetCertificateChainReq. Instead this function can internally start from offset 0 and read until it gets to the end.

Reply:
Yes, moved it into the client function